classes/exercises.py

```diff
@@ -26,3 +26,39 @@
 #    because we already know that all of them should be 60.
 
 # 2.1. Create a `Equilateral` class method that receives a side length and calculates the triangle area.
+
+class Triangle(object):
+
+    number_of_sides = 3
+
+    def __init__(self, angle1, angle2,angle3):
+        self.angle1 = angle1
+        self.angle2 = angle2
+        self.angle3 = angle3
+
+    def check_angles(self):
+        if sum([self.angle1,self.angle2,self.angle3]) == 180:
+            return True
+        return False 

```

Comment:
why create an if statement to return the if condition?
```suggestion
        return sum([self.angle1,self.angle2,self.angle3]) == 180

```